addons/stock_dropshipping/views/sale_order_views.xml

```diff
@@ -5,28 +5,17 @@
             <field name="name">sale.order.form.sale.dropshipping</field>
             <field name="model">sale.order</field>
             <field name="inherit_id" ref="sale.view_order_form"/>
-            <field name="groups_id" eval="[(4, ref('purchase.group_purchase_user'))]"/>
             <field name="arch" type="xml">
-                <xpath expr="//page/field[@name='order_line']/form//field[@name='product_updatable']" position="after">
-                    <field name="purchase_line_count" invisible="1"/>
-                </xpath>
-                <xpath expr="//page/field[@name='order_line']/form//field[@name='product_id']" position="attributes">
-                   <attribute name="attrs">{'readonly': ['|', ('product_updatable', '=', False), ('purchase_line_count', '&gt;', 0)], 'required': [('display_type', '=', False)],}</attribute>
-                </xpath>
-                <xpath expr="//page/field[@name='order_line']/tree/field[@name='product_updatable']" position="after">
-                    <field name="purchase_line_count" invisible="1"/>
-                </xpath>
-                <xpath expr="//page/field[@name='order_line']/tree/field[@name='product_id']" position="attributes">
-                    <attribute name="attrs">{'readonly': ['|', ('product_updatable', '=', False), ('purchase_line_count', '&gt;', 0)], 'required': [('display_type', '=', False)],}</attribute>
-                </xpath>
                 <xpath expr="//button[@name='action_view_delivery']" position="before">
```

Comment:
while modifying nearly the whole view, the xpath could be converted to `<button name="..." position="before">`.
IMHO, it's cleaner and clearer :)

Reply:
It is, easier, but increase the diff for no reason than just nitpicking. So for the moment I will leave it like that.